frontend/README.md

```diff
@@ -1,117 +1,75 @@
-# Resilience Atlas web app
+# Resilience Atlas - Front-end
 
 *This version is based on the repository: [https://github.com/ConservationInternational/resilienceatlas-react](https://github.com/ConservationInternational/resilienceatlas-react)*
 
-This is the web app powering
-[resilienceatlas.org](http://www.resilienceatlas.org)
+The front-end application of the Resilience Atlas platform is built using the following resources:
 
+- [React](https://reactjs.org/) as a UI library
+- [Next.js](https://nextjs.org/) as a framework
+- [Sass](https://sass-lang.com/) as a CSS language
+- [Foundation](https://get.foundation/) as a styles framework
+- [Redux](https://redux.js.org/) as a state manager
+- [Leaflet](https://leafletjs.com/) and [CARTO.js](https://carto.com/developers/carto-js/) as mapping technologies
 
-## Requirements:
+## Quick start
 
-- NodeJs 11.9.0+ [How to install](https://nodejs.org/download/)
-- npm 6.9.0+
-- yarn 1.13.0+ (you can use npm instead)
+In order to start modifying the app, please make sure to correctly configure your workstation:
 
-## Start here if your machine already has Requirements Installed, If you need help with installation, scroll to bottom
-Install project dependencies:
+1. Make sure you you have [Node.js](https://nodejs.org/en/) installed
+2. (Optional) Install [NVM](https://github.com/nvm-sh/nvm) to manage your different Node.js versions
+3. (Optional) Use [Visual Studio Code](https://code.visualstudio.com/) as a text editor to benefit from automatic type checking
+4. Configure your text editor with the [Prettier](https://prettier.io/), [ESLint](https://eslint.org/) and [EditorConfig](https://editorconfig.org/) plugins
+5. (Optional) Configure your editor to “format [code] on save” with ESLint and Prettier
+6. Use the correct Node.js version for this app by running `nvm use`; if you didn't install NVM (step 2), then manually install the Node.js version described in `.nvmrc`
+7. Install the dependencies: `yarn`
+8. Create a `.env` file at the root of the project by copying `.env.example` and giving a value for each of the variables (see next section for details)
+9. Run the server: `yarn dev`
 
-    yarn
+You can access a hot-reloaded version of the app on [http://localhost:3000](http://localhost:3000).
 
-## Usage
+## Environment variables
 
-Before running the application, you need to configure it by copying `.env.example` to `.env` and setting the appropriate values where needed.
+The application is configured via environment variables stored in a `.env` file that must be placed at the root of the project. You can create one by copying `.env.example` and setting a value for each key.
 
-To start the application, run:
+Below is a description of each of the keys.
 
-    yarn dev
+| Variable               | Description                                                                                                                                       |
```

Comment:
@davidsingal I think we could reduce the number and rename the environment variables. Is this something you'll do in #89?

Reply:
Yes, only `NEXT_PUBLIC_API_HOST` will be required

Reply:
I removed the other two environment variables.

Make sure to replace `REACT_APP_SITE` in your branch and to update https://github.com/ConservationInternational/resilienceatlas/blob/develop/.github/workflows/frontend_test.yml#L63-L65.

Reply:
Updated!